packages/cli/src/standalone-impl.ts

```diff
@@ -1,124 +1,19 @@
 import {
-  SequentialDataAccessor,
-  DataAccessor,
   PipelineMeta,
-  ImageDataSourceMeta,
-  TableDataSourceMeta,
-  SequentialDataSourceApi,
-  DataSourceApi,
-  Sample,
-  Runtime,
   TaskType,
-  DefaultType,
-  ProgressInfo,
-  pipelineAsync,
-  seedrandom,
-  generateId
+  ProgressInfo
 } from '@pipcook/core';
+import { pipelineAsync } from './utils';
+import { DefaultDataSource, DefaultRuntime } from '@pipcook/costa';
 import * as fs from 'fs-extra';
 import * as path from 'path';
 
-export class DataAccessorImpl<T> implements DataAccessor<T> {
-  size: number;
-  randomIndex: number;
-  randomArray: Array<number>;
+export class StandaloneImpl implements DefaultRuntime {
   constructor(
-    private dataAccessor: SequentialDataAccessor<T>
-  ) {}
-  init(size: number, seed?: string): void {
-    this.size = size;
-    this.randomIndex = 0;
-    this.randomArray = this.createRandom(this.size, seed);
-  }
-  next(): Promise<Sample<T> | null> {
-    return this.dataAccessor.next();
-  }
-  nextBatch(batchSize: number): Promise<Array<Sample<T>> | null> {
-    return this.dataAccessor.nextBatch(batchSize);
-  }
-  nextRandom(): Promise<Sample<T> | null> {
```

Comment:
Where do these methods go?

Reply:
`DataCook.dataset.shuffle()` will instead of it.